server/src/routes/auth.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { signup } = require("../controllers/auth");
+const { validateSignupRequest, isRequestValidated } = require("../validators/auth");
+const router = express.Router();
+
+router.post('/signup',validateSignupRequest, isRequestValidated, signup);
```

Comment:
In the example.route.js I have written to write the a short description of the route which you should also add for the frontend developers who will be referring the api before using it

Reply:
ok will add

Reply:
You missed out on this too, kindly look carefully and do the needful. Change the file name to auth.route.js as well